src/dev/konnector.md

```diff
@@ -508,6 +508,33 @@ Cozy will soon have a store for connectors and you will be able to publish conne
 at the moment, you need to declare your new connector on the [cozy forum](https://forum.cozy.io).
 The Cozy team will review your code and add your connector to the [Cozy Collect] application.
 
+To make the connector available more quickly for all cozys, you can follow this few steps of 
+pre-packaging:
+
+### Icon
+
+You need to push an icon in `assets/icon.png`. Please respect this rules :
+ - Square icon, possibly a png or svg
+ - Try the apple store icon if needed
+ - Set the correct name in manifest
```

Comment:
Name the file
> Set the correct name in `manifest.konnector`

Reply:
Oh, maybe we can cover this step in the `Manifest.konnector` chapter? And remove this line.